README.md

```diff
@@ -24,9 +25,11 @@ Compensated makes it easier to get paid by your users. It listens for events fro
 Compensated's architecture draws from prior art such as [griddler](https://github.com/thoughtbot/griddler), [omniauth](https://github.com/omniauth/omniauth) and [expressjs](https://expressjs.com/). The goal of the design is to enable functionality without imposing demands on structures.
 
 ## Contributing
-See [CONTRIBUTING.md](CONTRIBUTING.md)
 
+To get started, make sure you have [rbenv](https://github.com/rbenv/rbenv) installed, clone the repository and run `bin/setup-matrix` then run `bin/test-matrix`.
```

Comment:
Nit: It may be worth calling out directly that if there are errors running either of the scripts; they are encouraged to read the content of the files and use them to guide their debugging.